applications/conceptual-model-editor/src/app/diagram/dialog/select-classes-dialog.tsx

```diff
@@ -0,0 +1,117 @@
+import { useCallback, useState } from "react";
+import { useModelGraphContext } from "../context/model-context";
+import { isSemanticModelClass } from "@dataspecer/core-v2/semantic-model/concepts";
+import { useBaseDialog } from "../components/base-dialog";
+import { useConfigurationContext } from "../context/configuration-context";
+import { CreateButton } from "../components/dialog/buttons/create-button";
+import { CancelButton } from "../components/dialog/buttons/cancel-button";
+import { t } from "../application/";
+import { isSemanticModelClassUsage } from "@dataspecer/core-v2/semantic-model/usage/concepts";
+import { type EntityDetailSupportedType, EntityProxy } from "../util/detail-utils";
+import { tailwindColorToHex } from "~/app/utils/color-utils";
+
+const getDefaultColor = () => {
+    return "#069420";
```

Comment:
This should be part of the configuration.

Reply:
I agree, that being said I stole it from [here](https://github.com/mff-uk/dataspecer/blob/21aeaf55e12c4744de12cc66dc86d409823e6d9e/applications/conceptual-model-editor/src/app/diagram/catalog/components/model-entities-header.tsx#L9-L22) and I haven't quite checked code to see what is the reason for having 4 default colors, so for now I just use local "copy" of the default color.